vagrant/provision-master.sh

```diff
@@ -33,7 +33,10 @@ popd
 # Initialize certificates
 echo "Generating certs"
 pushd /vagrant
-  CERT_DIR=`pwd`/openshift.local.certificates
+  SERVER_CONFIG_DIR="`pwd`/openshift.local.config"
+  MASTER_CONFIG_DIR="${SERVER_CONFIG_DIR}/master"
+  NODE_CONFIG_DIR="${SERVER_CONFIG_DIR}/node"
+  CERT_DIR="${MASTER_CONFIG_DIR}/certificates"
```

Comment:
certs created to this dir by `create-master-certs` won't be found by `openshift start` below